common/src/main/java/com/hedera/block/common/utils/FileUtilities.java

```diff
@@ -143,20 +146,22 @@ public static void createPathIfNotExists(
     /**
      * Read a GZIP file and return the content as a byte array.
      * <p>
-     * This method is _unsafe_ because it reads the entire file content into
-     * a single byte array, which can cause memory issues, and may fail if the
-     * file contains a large amount of data.
+     * This method is _unsafe_ because it reads the entire file content into a single byte array,
+     * which can cause memory issues, and may fail if the file contains a large amount of data.
      *
      * @param filePath Path to the GZIP file.
+     *
      * @return byte array containing the _uncompressed_ content of the GZIP file.
-     * @throws IOException if unable to read the file.
-     * @throws OutOfMemoryError if a byte array large enough to contain the
-     *     file contents cannot be allocated (either because it exceeds MAX_INT
-     *     bytes or exceeds available heap memory).
+     *
+     * @throws IOException      if unable to read the file.
+     * @throws OutOfMemoryError if a byte array large enough to contain the file contents cannot be
+     *                          allocated (either because it exceeds MAX_INT bytes or exceeds
+     *                          available heap memory).
      */
     public static byte[] readGzipFileUnsafe(@NonNull final Path filePath) throws IOException {
         Objects.requireNonNull(filePath);
-        try (final var gzipInputStream = new GZIPInputStream(Files.newInputStream(filePath))) {
+        try (final GZIPInputStream gzipInputStream =
+                new GZIPInputStream(Files.newInputStream(filePath))) {
```

Comment:
This was correct as it stood, there is no reason to reformat the line.
The use of `var` was deliberate to avoid an unnecessary line break.
The only reformatting we should be doing is via `./gradlew spotlessApply` so we don't
have random preferences creating back-and-forth warring reformats.

Reply:
✅ Personally I would never use `var`, but since that is the accepted norm and if it saves us the line break, I take a note. This is reversed now. And as for the reformatting, I do only use spotless, but looks like my IDE also applied reformatting and I thought it was from spotless on files I have changed, this should no longer be the case.

Reply:
I generally agree on `var`, except when it makes a major improvement in readability, as here.